### DISCUSSION TITLE
Are we supposed to add .content-collections to .gitignore?

### DISCUSSION THREAD

**Discussion by rossmeissl** [sdorra/content-collections#223] [Q&A] [answered]
Are we supposed to add .content-collections to .gitignore?

**Comment by sdorra** [comment, marked as answer]
Yes, the files in the .content-collections directory are always regenerated and can be safely ignored by Git.

**Comment by rossmeissl** [comment]
@sdorra Thanks! https://github.com/sdorra/content-collections/pull/236
